torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -332,13 +332,14 @@ LLVMCodeGen::LLVMCodeGen(
     const std::vector<BufferArg>& args,
     at::Device device,
     const std::string& kernel_func_name,
+    const bool pre_alloc,
     Dtype dtype,
     c10::optional<std::string> triple,
     c10::optional<std::string> cpu,
     c10::optional<std::string> attrs)
-    : CodeGen(stmt, args, device, kernel_func_name) {
+    : CodeGen(stmt, args, device, kernel_func_name, pre_alloc) {
   impl_ = std::make_unique<LLVMCodeGenImpl>(
-      this->stmt(), args, device, dtype, triple, cpu, attrs);
+      this->stmt(), this->buffer_args(), device, dtype, triple, cpu, attrs);
```

Comment:
Why is this change needed?

Reply:
Yes it's needed. The buffer args will be updated in the constructor of its parent (CodeGen).